README.md

```diff
@@ -34,12 +34,14 @@ load("@bazel_bootlin//toolchains:toolchains.bzl", "bootlin_toolchain_deps")
 
 bootlin_toolchain_deps(
     architecture = "x86-64-core-i7",
+    cstdlib = "gnu",
```

Comment:
I think you are mixing up `gnu` and `glibc`  `linux-gnu` describes the os environment I think right?  Maybe we remove references to `gnu` entirely?

Reply:
Done